nevergrad/optimization/base.py

```diff
@@ -165,6 +171,53 @@ def num_tell_not_asked(self) -> int:
         """
         return self._num_tell_not_asked
 
+    def penalty(self, candidate: p.Parameter):
```

Comment:
this should be hidden, please append with `_`.
I'll probably update this with a specific class anyway

Reply:
Done.